src/hooks/useConnectorTags.ts

```diff
@@ -0,0 +1,48 @@
+import { TABLES } from 'services/supabase';
+import { OpenGraph } from 'types';
+import { useQuery, useSelect } from './supabase-swr/';
+
+export interface ConnectorTagQuery {
+    connectors: {
+        detail: string;
+        image_name: string;
+        open_graph: OpenGraph;
+    };
+    id: string;
+    connector_id: string;
+    image_tag: string;
+    protocol: string;
+    title: string;
+}
+export const CONNECTOR_TAG_QUERY = `
+    id,
+    connector_id,
+    image_tag,
+    protocol,
+    endpoint_spec_schema->>title,
+    connectors(detail, image_name, open_graph)
+`;
+
+function useConnectorTags(protocol: string | null) {
```

Comment:
New hook to fetch connector tags. This is like 95% the same code as what was living in both the create pages.